CBLClient/Apps/CBLTestServer-Dotnet/TestServer/ReplicatorConfigurationMethods.cs

```diff
@@ -372,4 +400,207 @@ private static bool _replicator_access_revoked_filter_callback(Document document
             });
         }
     }
-}
+
+    internal sealed class LocalWinsCustomConflictResolver : IConflictResolver
+    {
+        Document IConflictResolver.Resolve(Conflict conflict)
+        {
+            Document localDoc = conflict.LocalDocument;
+            Document remoteDoc = conflict.RemoteDocument;
+            string docId = conflict.DocumentID;
```

Comment:
need to make sure either the local doc or the remote doc is not null, if any of the doc happens to be a null object, then this is not appropriate scenario.

Reply:
same as above.

Reply:
Added null check